doc/ansible/user-guide.md

```diff
@@ -395,6 +395,26 @@ kubectl logs deployment/memcached-operator -c operator
 The `ansible` logs contain all of the information about the Ansible run and will make it much easier to debug issues within your Ansible tasks,
 whereas the `operator` logs will contain much more detailed information about the Ansible Operator's internals and interface with Kubernetes.
 
+### Additional Ansible debug
+
+Occasionally while developing additional debug in the Operator logs is nice to have. To enable Ansible debug output, ie `-vvvv`.
+Add the following to the Operator manifest.
+
+```yaml
+          env:
+           ...
+           - name: ANSIBLE_VERBOSITY
+             value: "4"
+```
+
+Additionally, to view formatted output, you can run,
+
+```
+$  oc exec <operator-pod> -- cat /tmp/ansible-operator/runner/<crd group>/v1alpha1/<kind>/<namespace>/<cr>/artifacts/latest/stdout
```

Comment:
I would probably remove this, or at least note again that if you want to you can aggregate all those logs with a sidecar we built: https://github.com/operator-framework/operator-sdk/pull/1352/files#diff-2a1f94c1f2f65c1266ac5d5d482009c7R391